birdhouse/env.local.example

```diff
@@ -16,7 +16,7 @@ export TOMCAT_NCWMS_PASSWORD=ncwmspass
 export SUPPORT_EMAIL=helpdesk@example.com
 export CMIP5_THREDDS_ROOT=birdhouse/CMIP5/CCCMA
 export JUPYTERHUB_ADMIN_USERS="{'admin'}"  # python set syntax
-export JUPYTERHUB_USERS_PASS="admin:admin public:public"  # space separated 'username:passwd' format
+export JUPYTERHUB_USER_DATA_DIR=/data/jupyterhub_user_data
```

Comment:
This should be an optional var instead of mandatory vars.  It's not like a password were user is much encouraged to change the default value.

Optional var do not break the deployment if `env.local` do not contain it (much easier for the auto deployment).  New vars introduced should be optional with sensible default.

Mandatory is reserved when we really want user to take action, like to set a new password, else deployment won't work.